packages/quantic/force-app/main/default/lwc/quanticUtils/__tests__/quanticUtils.test.js

```diff
@@ -220,4 +221,71 @@ describe('c/quanticUtils', () => {
       });
     });
   });
+
+  describe('extractTextToHighlight', () => {
```

Comment:
Is there a reason why this isn't simply inside Headless instead of being duplicated between Atomic and Quantic?